### ISSUE TITLE
A11y_VSCode_SidebarBarOptions_AccessingTheExplorersidebaroptionsandCreatinganewfile_ScreenReader: The “New File”, “New Folder”, etc. buttons are announced with the state as selected in MAC.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode#76981]
**Environment Details:**
VSCode Version : 1.35.6 (user setup)
OS Version : MAC OS Mojave, V 10.14.5, Mac Mini Late2012

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : Voiceover

**Repro Steps:**
1.	Pre-req: Create an empty folder on desktop
2.	Open Visual studio code editor.
3.	Using "Ctrl+Shift+E" key navigate to the Explorer side view bar option.
4.	Using "Tab" key navigate to explorer side bar options.
5.	Arrive at the "Open Folder" button and hit "Enter".
6.	Select the folder created under "Pre-req".
7.	Start screen reader.
8.	Using "Tab" key navigate to and select "New File" option.
9.	Listen if the state announced is correct or not just on navigating through the controls.

**Actual:**
In MAC when screen reader users navigate to the “New file” “New Folder”, etc. button the state as selected is announced which in incorrect as the selected state should be announced when the user activates the button not on navigation.

**Expected:**
The “New File”, “New Folder”, etc. buttons should not be announced with the selected state when screen reader users are navigating through these controls.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If the state of the interactive elements are not announced properly to the screen reader users then the users might not know how to interact with them.

**MAS Reference:**
MAS 1.3.1 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

**Attachment for Reference:**
[VSCode_SidebarBarOptions_AccessingTheExplorersidebaroptionsandCreatinganewfile_ScreenReader Incorrect state announced in MAC..pptx](https://github.com/microsoft/vscode/files/3373051/VSCode_SidebarBarOptions_AccessingTheExplorersidebaroptionsandCreatinganewfile_ScreenReader.Incorrect.state.announced.in.MAC.pptx)

**Comment by isidorn** [comment]
Works just fine
![hello](https://user-images.githubusercontent.com/1926584/60954471-4f29a400-a2ff-11e9-87c8-b5fa9e275c78.gif)
